### ISSUE TITLE
Error on constructor

### ISSUE THREAD

**Issue by LorhanSohaky** [jdalrymple/gitbeaker#686]
**Description**
I have an application in Vue and when trying to run the contructor, it gives the following error.

![image](https://user-images.githubusercontent.com/16273730/78957157-e4d6a780-7aba-11ea-95ee-68e4a31748ef.png)

```
_gitbeaker_browser__WEBPACK_IMPORTED_MODULE_6__.Gitlab is not a constructor
    at new GitLab (gitlab.js:7)
    at _callee5$ (AfterLogin.vue:369)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:274)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
    at asyncToGenerator.js:32
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21
    at VueComponent.getUser (AfterLogin.vue:361)
    at VueComponent.getInfo (AfterLogin.vue:304)
    at VueComponent.mounted (AfterLogin.vue:193)
    at invokeWithErrorHandling (vue.runtime.esm.js:1854)
    at callHook (vue.runtime.esm.js:4219)
    at Object.insert (vue.runtime.esm.js:3139)
    at invokeInsertHook (vue.runtime.esm.js:6346)
    at Vue.patch [as __patch__] (vue.runtime.esm.js:6565)
    at Vue._update (vue.runtime.esm.js:3945)
    at Vue.updateComponent (vue.runtime.esm.js:4066)
    at Watcher.get (vue.runtime.esm.js:4479)
    at new Watcher (vue.runtime.esm.js:4468)
    at mountComponent (vue.runtime.esm.js:4073)
    at Vue.push../node_modules/vue/dist/vue.runtime.esm.js.Vue.$mount (vue.runtime.esm.js:8415)
    at Module../src/main.js (main.js:53)
    at __webpack_require__ (bootstrap:784)
    at fn (bootstrap:150)
    at Object.1 (Settings.vue?66a4:1)
    at __webpack_require__ (bootstrap:784)
    at checkDeferredModules (bootstrap:45)
    at bootstrap:860
    at bootstrap:860
```

**Steps to reproduce**
## Dependencies
node: v10
```json
  "vue": "^2.6.11",
  "@gitbeaker/browser": "^17.0.1",
```
## Code
```vue
import { Gitlab } from "@gitbeaker/browser";

const token = "<token>";
try {
  const api = new Gitlab({
    token
  });
} catch(err) {
  console.error(err);
}
```

**Comment by jdalrymple** [comment]
Could you provide a sample repo with this?

**Comment by LorhanSohaky** [comment]
https://codesandbox.io/s/crimson-sea-cf5uk?file=/src/gitlab.js

**Comment by jdalrymple** [comment]
Should be good with this latest release. Give it a look in a few minutes!

**Comment by jetersen** [comment]
Yes: https://codesandbox.io/s/wizardly-germain-whvr6?file=/src/gitlab.js switching to v19.0 makes it work

**Comment by LorhanSohaky** [comment]
Thanks!!!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`22.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/22.0.0) :rocket:
